streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorContextImplTest.java

```diff
@@ -235,7 +383,7 @@ public void close() {
         processor.init(context);
     }
 
-    private void checkStateStoreMethods(final StateStore store) {
+    private void checkGlobalStateStoreMethods(final StateStore store) {
```

Comment:
The only difference between local and global should be flushing -- might better to test `flush` in "main" part and use one method to check all other common ones? Rename method to `verifyStoreCannotBeInitializedOrClosed`?